src/index.html

```diff
@@ -1,17 +1,164 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="groupField">
+        <legend>Personal information:</legend>
+        <label for="surname" class="form-field">
+          Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label for="name" class="form-field">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            required 
+            autocomplete="off"
+          >
+        </label>
+      
+        <label for="age" class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+      
+        <label for="dateOfBirth" class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            value="mm/dd/yyyy"
+          >
+        </label>
+      
+        <label for="terms">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
```

Comment:
You don't need the for attribute if you wrap the element into a label. This label doesn't work correctly now. The checkbox should change state on the text click. Fix it for all labels, please 
```suggestion
        <label>
          I accept the term of the agreement
          <input type="checkbox" name="terms">
```